src/BrancherServer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hypernode\DeployConfiguration;
+
+class BrancherServer extends Server
+{
+    public function getLabels(): array
+    {
+        return $this->getOptions()[self::OPTION_HN_BRANCHER_LABELS] ?? [];
+    }
+
+    /**
+     * @param string[] $labels Labels to be applied to the brancher node
+     * @return $this
+     */
+    public function setLabels(array $labels): self
+    {
+        $this->setOption(self::OPTION_HN_BRANCHER_LABELS, $labels);
```

Comment:
Do you think it's wise to add input validation in the client here or should we let the API validate the data?

Reply:
Yes I think so, since this is a very light component, let's add the validation in hypernode-deploy